src/assets/TeamMembersData.json

```diff
@@ -0,0 +1,130 @@
+{
+    "members": [
+        {
+            "name": "1",
+            "designation": "Founder",
+            "img": "https://source.unsplash.com/random/200x200",
+            "githublink": "github.com/abcdef",
+            "linkedin": "linkedin.com/in/abcdef",
+            "phonenNo": "9898989898",
+            "emailid": "abcdef@gmail.com"
+        },
```

Comment:
name, designation, imageUrl, githubProfile, linkedinProfile, phoneNumber, emailId might be better names.

Reply:
Okay,I will change them